src/PackageUtilities.js

```diff
@@ -141,8 +141,9 @@ export default class PackageUtilities {
     if (!Array.isArray(glob)) glob = [glob];
 
     const maybeNegate = negate ? (v) => !v : (v) => v;
+    const boolReducer = negate ? Array.prototype.every : Array.prototype.some;
 
-    return glob.some((glob) => maybeNegate(minimatch(pkg.name, glob)));
+    return boolReducer.call(glob, (glob) => maybeNegate(minimatch(pkg.name, glob)));
```

Comment:
I thought about doing that but didn't want to rock the boat in terms of the existing code. Agreed, that's much simpler, so I'll update the PR with that refactor. 👍